### ISSUE TITLE
6.0.0-beta.8 ignores inline style & template, along with scss config.

### ISSUE THREAD

**Issue by robertbaker** [angular/angular-cli#10105]
### Versions
```
Angular CLI: 6.0.0-beta.8
Node: 9.9.0
OS: win32 x64

```

### Repro steps
`ng new test-app --routing -it -is -si --style=scss' <- this generates properly
`ng g c test`

### Observed behavior
```
CREATE projects/pizza-client/src/app/test/test.component.html (23 bytes)
CREATE projects/pizza-client/src/app/test/test.component.spec.ts (614 bytes)
CREATE projects/pizza-client/src/app/test/test.component.ts (257 bytes)
CREATE projects/pizza-client/src/app/test/test.component.css (0 bytes)
UPDATE projects/pizza-client/src/app/app.module.ts (467 bytes)
```

### Desired behavior
```
CREATE projects/pizza-client/src/app/test/test.component.spec.ts (614 bytes)
CREATE projects/pizza-client/src/app/test/test.component.ts (257 bytes)
UPDATE projects/pizza-client/src/app/app.module.ts (467 bytes)
```

### Mention any other details that might be useful (optional)
I would assume other configurations are affected, such has --module=something or `--minimal` /` --skip-tests`

This appears to have broken from the migration from angular-cli.json to the new angular.json,
there is no defaults section

**Comment by maroy1986** [comment]
I've also it that issue with SCSS not being taken into account when using `ng g c WhateverComponentName`.

Is this something we missed when migrating from the file to the new one?

**Comment by mischkl** [comment]
On a newly-created project, I haven't found any way to ensure that new components are created with SCSS styles. Even if the project is created using `--style=scss` it makes no difference. It seems like support for setting the default styleExt within the project is completely missing!

**Comment by maroy1986** [comment]
The RC3 version didn't fix it either.

**Comment by mruknowme** [comment]
I tried --style=sass and --prefix=myprfx, none of them work. Style applied only for the main styles file and for the 'app.component', not for new generated ones.

I am surprised that nobody has fixed it since like beta 5 or so...

**Comment by maroy1986** [comment]
RC4 is out, not yet fixed.

**Comment by maroy1986** [comment]
RC5 fixed it. It's now working properly when doing `ng new AppTest --style=scss`.

**Comment by cyrilletuzi** [comment]
This is already fixed by angular/devkit#666, released in RC5.

But there is an issue with option with an uppercase letter: #10371
